acceptance/lib/beaker/acceptance/install_utils.rb

```diff
@@ -0,0 +1,58 @@
+module Beaker
+  module Acceptance
+    module InstallUtils
+
+      PLATFORM_PATTERNS = {
+        :redhat        => /fedora|el|centos/,
+        :debian        => /debian|ubuntu/,
+        :debian_ruby18 => /debian|ubuntu-lucid|ubuntu-precise/,
+        :solaris_10    => /solaris-10/,
+        :solaris_11    => /solaris-11/,
+        :windows       => /windows/,
+        :sles          => /sles/,
+      }.freeze
+
+      # Installs packages on the hosts.
+      #
+      # @param hosts [Array<Host>] Array of hosts to install packages to.
+      # @param package_hash [Hash{Symbol=>Array<String,Array<String,String>>}]
+      #   Keys should be a symbol for a platform in PLATFORM_PATTERNS.  Values
+      #   should be an array of package names to install, or of two element
+      #   arrays where a[0] is the command we expect to find on the platform
+      #   and a[1] is the package name (when they are different).
+      # @param options [Hash{Symbol=>Boolean}]
+      # @option options [Boolean] :check_if_exists First check to see if
+      #   command is present before installing package.  (Default false)
+      # @return true
+      def install_packages_on(hosts, package_hash, options = {})
+        return true if hosts == nil
+        check_if_exists = options[:check_if_exists]
+        hosts = [hosts] unless hosts.kind_of?(Array)
+        hosts.each do |host|
```

Comment:
Ah, gotcha.  Don't mind me then.  Those are my run-of-the-mill "if we're adding new ruby code" sorts of comments.